index.bs

```diff
@@ -369,6 +448,110 @@ Report long tasks {#report-long-tasks}
         1. [=Queue the PerformanceEntry=] |newEntry|.
 </div>
 
+
+Frame Timing Info {#sec-frame-timing-info}
+------------------------------------------
+<dfn export>frame timing info</dfn> is a [=struct=] used as a bookkeeping detail by the long animation frame algorithms.
+It has the following [=struct/items=]:
+
+<dl dfn-for="frame timing info">
+    : <dfn>start time</dfn>
+    : <dfn>current task start time</dfn>
+    : <dfn>update the rendering start time</dfn>
+    : <dfn>style and layout start time</dfn>
+    : <dfn>longest task duration</dfn>
+    : <dfn>end time</dfn>
+    :: A {{DOMHighResTimeStamp}}, initially 0.
+</dl>
+
+A {{Document}} has a null-or-[=frame timing info=] <dfn>current frame timing info</dfn>, initially null.
```

Comment:
Can we just append a "null-or-" to random structs without defining the whole thing?

Reply:
I don't understand

Reply:
It seems like your defining a new type named "null-or-frame timing info".  But I see a similar pattern in Fetch, only without the "-". Maybe drop them?